### ISSUE TITLE
The common/config/chartserver/env is rendered incorrect when regionendpoint is empty

### ISSUE THREAD

**Issue by stonezdj** [goharbor/harbor#7688]
```yml
storage_service:                                                                                                      
  s3:                                                                                                                                                                                                                                       
    accesskey: xxxxx                                                                                                                                                                                                         
    secretkey: xxxxxxx                                                                                                                                                                                     
    region: us-west-2                                                                                                                                                                                                                       
    regionendpoint:                                                                                                                                                                                                                         
    bucket: photon-harbor-registry                                                                                                                                                                                                          
    encrypt: true                                                                                                                                                                                                                           
    secure: true                                                                                                                                                                                                                            
    v4auth: true                                                                                                                                                                                                                            
    chunksize: 5242880         
```

The env is render incorrect 
```shell
STORAGE_AMAZON_ENDPOINT=None
```
It should be rendered as empty string
```shell
 STORAGE_AMAZON_ENDPOINT=    
```

The same issue exist in common/config/registry/config.yml

sometimes the root directory also rendered to None, actually, it is emtpy string, and the True should be true with same format, because I am not sure registry can consider it as true?
```yml
  s3:                                                                                                                 
    rootdirectory: None                                                                                               
    secure: True                                                                                                      
    accesskey: SFGKZMTXNOCQN7ZQW2QA                                                                                   
    bucket: zdjtest                                                                                                   
    v4auth: True                                                                                                      
    secretkey: 0Wdmv5yPRKj3SWLgIL812BZNzVTv+O1gqfH+tT9f                                                               
    regionendpoint: https://10.161.51.79                                                                              
    region: generic                                                                                                   
    chunksize: 5242880  
```

**Comment by ninjadq** [comment]
this is fixed in #7639

**Comment by stonezdj** [comment]
This issue is can produced in build 1486
